locales/pl-PL.json

```diff
@@ -175,11 +175,11 @@
         "chodnik",
         "ścieżka rowerowa",
         "górski szlak rowerowy",
-        "the crosswalk"
+        "przejście dla pieszych"
       ],
       "metric_lengths": [
         "<KILOMETERS> km",
-        "1 km",
+        "1 kilometr",
         "<METERS> m",
```

Comment:
since t his is verbal we probably shouldnt have any abbreviations.. i guess we'll get it later